### PR TITLE
secapr update

### PR DESCRIPTION
**Pull request by tobiashofmann88**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by tobiashofmann88** [comment]
Not sure what is going wrong, by now I tried everything I can come up with.
The current error is the following:
> Can't build /opt/recipe due to unsatisfiable dependencies:
> The following specifications were found to be in conflict:
>   - secapr 1.1.7 np114py27r3.4.1_0

**Comment by marcelm** [comment]
Only guessing now, but try to remove the version specification from `numpy`.

Also, specify the same versions for `pandas` and `biopython` in both the `run:` and `build:` requirements sections.

Do only one of the above at a time.

**Comment by tobiashofmann88** [comment]
okay its working now!! Thanks for your help @marcelm!

**Comment by tobiashofmann88** [comment]
@bioconda/core: I think this recipe is now ready to be merged. Thanks!!

File: recipes/secapr/meta.yaml [hunk 1]
```diff
@@ -22,27 +25,28 @@ requirements:
...
+    - samtools ==0.1.19
+    - picard ==1.126
+    - seqtk >=1.0.82,<=1.2
+    - bwa >=0.7
+    - lastz ==1.0.2
+    - mafft >=7.2
+    - muscle ==3.8.31
+    - trimmomatic ==0.33
+    - abyss ==1.5.2
+    - fastqc >=0.11,<=0.12
```
Comment (marcelm) [inline recipes/secapr/meta.yaml]: You can express this as `fastqc 0.11*`

File: recipes/secapr/meta.yaml [hunk 2]
```diff
@@ -22,27 +25,28 @@ requirements:
...
-    - muscle
-    - cogent
-    - trimmomatic
-    - trinity
-    - abyss
-    - fastqc
-    - pandas
-    - numpy 
-
+    - biopython ==1.68
```
[the remark this answers is not in the document]
Comment (marcelm) [reply recipes/secapr/meta.yaml]: To Tobias defense: Yes, these are the versions that `secapr` has been tested with. I think this needs to be allowed for applications.

File: recipes/secapr/meta.yaml (hunk 1 above)
Comment (mbargull) [reply recipes/secapr/meta.yaml]: Yes, in the general case this should even be either `fastqc 0.11.*` or `fastqc >=0.11,<0.12.0a0` -- if (hypothetically) there where an alpha version `0.12.a1`, that one would be considered `<0.12`.

**Comment by tobiashofmann88** [comment]
I implemented all suggested changes and I hope it's good to go now!
Thanks a lot for your help!

**Comment by marcelm** [comment]
Tobias wants to use this package at a workshop where he’s teaching today, so I’m taking the liberty of merging this now as it seems all suggestions have been addressed. There can always be another PR for any remaining fixes.

**Pull request merged**